db/migrate/20200411005517_add_status_to_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddStatusToOrders < ActiveRecord::Migration[5.1]
+  def change
+    add_column :orders, :status, :string, default: 'pending'
```

Comment:
wondering if its better to have the db use integers here 
so the default would be 0